task-sdk/tests/task_sdk/execution_time/test_supervisor.py

```diff
@@ -712,104 +709,56 @@ def test_overtime_handling(
             mock_kill.assert_not_called()
             mock_logger.warning.assert_not_called()
 
-
-class TestListenerOvertime:
```

Comment:
I deleted this test entirely as it was a) slow, b) failing with the new log message changes in a not-meaningful way, and c) duplicating the `test_overtime_handling` test just above.

(It was testing specifically the behaviour of Listeners in overtime, but overtime-is-overtime, it's all handled in the task subprocess and makes no difference to the supervisor)